pkgs/by-name/ze/zerocool_ttf/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "zerocool_ttf";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://web.archive.org/web/20250619041906/https://dl.dafont.com/dl/?f=zero_cool";
+    hash = "sha256-wpB5YdOlYcRLSGou1Wjk+oRGrwSiiL+f6BQNp/VUQe4=";
+    name = "zero_cool.zip";
+  };
+
+  nativeBuildInputs = [ unzip ];
+
+  unpackPhase = ''
+    unzip "$src"
+  '';
```

How about using `fetchzip`?

```suggestion
  fetchzip,
}:

stdenvNoCC.mkDerivation {
  pname = "zerocool_ttf";
  version = "1.0";

  src = fetchzip {
    url = "https://web.archive.org/web/20250619041906/https://dl.dafont.com/dl/?f=zero_cool";
    hash = "sha256-w9RNgO+kxhoO3HxK+ESrSA6FGljDwJfxo8TIUZzlzpA=";
    extension = "zip";
    stripRoot = false;
  };
```